### PR TITLE
FIX: #86 add show Details

### PR DESCRIPTION
**Pull request by ac491**
# Description

#86 add show details in pdf actions
copy
Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x ] `./gradlew checkstyle`

# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java [hunk 1]
```diff
@@ -78,7 +82,10 @@ public void onBindViewHolder(@NonNull ViewFilesHolder holder, final int position
         Log.d("logs", "getItemCount: " + mFileList.size());
         // Extract file name from path
         final String fileName = mFileList.get(position).getPath();
-        String[] name = fileName.split("/");
+        final String[] name = fileName.split("/");
+        File file = mFileList.get(position);
+        final Date lastModDate = new Date(file.lastModified());
+        final int file_size = Integer.parseInt(String.valueOf(file.length() / 1024));
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java]: We have a function to get filesize from here. Use this https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/java/swati4star/createpdf/util/FileUtils.java#L151

File: app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java [hunk 2]
```diff
@@ -78,7 +82,10 @@ public void onBindViewHolder(@NonNull ViewFilesHolder holder, final int position
         Log.d("logs", "getItemCount: " + mFileList.size());
         // Extract file name from path
         final String fileName = mFileList.get(position).getPath();
-        String[] name = fileName.split("/");
+        final String[] name = fileName.split("/");
+        File file = mFileList.get(position);
+        final Date lastModDate = new Date(file.lastModified());
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java]: We have a function to get last modified date also : https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/java/swati4star/createpdf/util/FileUtils.java#L134

If the format returned by this function is different than yours, create a new one.

**Comment by ac491** [comment]
@Swati4star are any more changes required or should I squash the commits?

**Comment by Swati4star** [comment]
@ac491 you can squash the changes now.

**Pull request merged**